integration_tests/test_upgrade_recent.py

```diff
@@ -91,6 +91,25 @@ async def exec(c):
     balance = cli.balance(eth_to_bech32(receiver), denom)
     assert balance == balance_eth == transfer_amt
 
+    # test approve
+    approve_amt = 10
```

Comment:
can be less than https://github.com/MANTRA-Chain/mantrachain-e2e/blob/8f7b1cd844113ce619f58ac2c21008bb2d4f8e7d/integration_tests/test_upgrade_recent.py#L81